pmacApp/src/pmacController.cpp

```diff
@@ -3936,7 +3957,12 @@ asynStatus pmacController::updateCsAssignmentParameters() {
         if (uCsAxisNo != std::string::npos) {
           // there is a direct mapping, tell the CS axis it has this mapping
           int csAxisAssignmentNo = (int) uCsAxisNo + 1;
-          pCSControllers_[csNo]->pmacCSSetAxisDirectMapping(csAxisAssignmentNo, axis);
+          if(pCSControllers_[csNo] != NULL) {
+            pCSControllers_[csNo]->pmacCSSetAxisDirectMapping(csAxisAssignmentNo, axis);
+          }
+          else {
+            printf("Cs Controller Mapping is BAD\n");
```

Comment:
Use asynPrint to print error?

Reply:
Thanks, that was some debugging code that got left in (but I will keep it as a asynPrint)